### ISSUE TITLE
[Suggestion]: First Person mod Collab

### ISSUE THREAD

**Issue by RicoR3t** [Virtuoel/Pehkui#356]
### What is your suggestion?

At the current moment, when putting First person mod with Pehkui, the different sized entities have some bugs with the cameras Y. I think it would be cool to be able to play with the first-person mod along your mod. If no, it's good. You're the expert not me.

**Comment by Aceplante** [comment]
Yes, specifically with crouching. The players body leans excessively forward and you cant see down. It only seems to happen with your mod. I remember it not being an issue before, so was there an update recently?

**Comment by Aceplante** [comment]
I was told it was because your y shift overwrites theirs? Please fix?

**Comment by twisting-my-melon-man** [comment]
I believe this is still an issue 1.19.2 3.7.8
